sources/OVPSDK.swift

```diff
@@ -406,23 +406,27 @@ public struct OVPSDK {
         }
         
         func setupVRMWithNewCore() {
+            let createRequest: (URL) -> (URLRequest) = {
```

Comment:
This have to be placed closer to the actual call place.

Reply:
If I correctly understand there will be ONLY ONE such place.

Reply:
There are two places already